rules/manager.go

```diff
@@ -69,6 +69,11 @@ var (
 		Help:       "The duration for all evaluations to execute.",
 		Objectives: map[float64]float64{0.01: 0.001, 0.05: 0.005, 0.5: 0.05, 0.90: 0.01, 0.99: 0.001},
 	})
+	iterationsSkipped = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
We have the total number of iteratoins via the summary above. It might still be clearer to have a seperate `evaluator_iterations_total`. (And maybe name this one `evaluator_skipped_iterations_total`).